src/views/uploadEvidence.html

```diff
@@ -0,0 +1,77 @@
+<main id="content" role="main">
+
+  {{> common/betaBanner }}
+
+  {{> common/backLink }}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {{> common/errorSummary }}
+
+      <h1 id="page-heading" class="heading-large">{{ pageHeading }}</h1>
+
+      {{#if annotations.length}}
+      <h2 class="heading-small">
```

Comment:
Needs an ID and corresponding unit test to check for existence on the page

Reply:
Done